src/ResourceManagement/Azure.Fluent/Fluent.Tests/Common/TestHelper.cs

```diff
@@ -298,5 +298,42 @@ private static bool HandlerContains<T1>(DelegatingHandler handler)
                 && handler.InnerHandler is DelegatingHandler
                 && HandlerContains<T1>(handler.InnerHandler as DelegatingHandler)));
         }
+
+        public static void DeprovisionAgentInLinuxVM(string host, int port, string userName, string password)
+        {
+            if (HttpMockServer.Mode == HttpRecorderMode.Playback)
+            {
+                return;
+            }
+            //SSHShell shell = null;
```

Comment:
Fixing tests and implementing this, forgot to put comment on this in first PR. Now fixed.